cypress/e2e/po/detail/provisioning.cattle.io.cluster/cluster-detail.po.ts

```diff
@@ -21,11 +21,51 @@ export default abstract class ClusterManagerDetailPagePo extends PagePo {
     return this.self().find('.primaryheader h1');
   }
 
+  logsContainer() {
+    return this.self().find('.logs-container');
+  }
+
   kubectlCommandForImported() {
     return this.self().get('code').contains('--insecure');
   }
 
   machinePoolsList() {
     return new MachinePoolsListPo(this.self().find('[data-testid="sortable-table-list-container"]'));
   }
+
+  conditionsList() {
+    return new MachinePoolsListPo(this.self().find('[data-testid="sortable-table-list-container"]'));
+  }
+
+  referredToList() {
+    return new MachinePoolsListPo(this.self().find('[data-testid="sortable-table-list-container"]'));
+  }
```

Comment:
these should have their own PO file like `MachinePoolsListPo` (cypress/e2e/po/lists/machine-pools-list.po.ts)